bellman/src/domain/fft.rs

```diff
@@ -0,0 +1,177 @@
+use crate::{multicore, arith};
+use multicore::Worker;
+use arith::Group;
+
+use ff::{Field, ScalarEngine};
+
+pub fn run_optimal_fft<E,T>(a: &mut [T], worker: &Worker, omega: &E::Fr, log_n: u32) 
+where
+    E: ScalarEngine,
+    for <'a> T: Group<'a,E> 
+{
+    let log_cpus = worker.log_num_cpus();
+
+    if log_n <= log_cpus {
+        serial_fft(a, omega, log_n);
+    } else {
```

Comment:
Please stick to a consistent style of {. I don't mind either way, but you should commit to a coherent style rather than chopping and changing between the bracket on a newline and on the same line

Reply:
Can you give me an example of the inconsistency?

Reply:
Resolved, idiomatic Rust